server/routes/search_relevance_route_service.ts

```diff
@@ -258,13 +260,44 @@ const backendAction = (method, path) => {
 
       return res.ok({ body: response });
     } catch (err) {
-      console.error('Failed to call search-relevance APIs', err);
+      console.error('Failed to call search-relevance APIs', err); // Keep for full server-side logging
+
+      let clientMessage = err.message; // Default to the err.message from transport.request
+      let clientAttributesError = err.body?.error || err.message; // Default attributes error
+
+      // Check if the backend error body contains the specific message
+      // This is the crucial part that extracts the "Search Relevance Workbench is disabled"
+      if (err.body && typeof err.body === 'string' && err.body.includes(DISABLED_BACKEND_PLUGIN_MESSAGE)) {
```

Comment:
Another thought, does the backend give a HTTP status error that we could check instead of string matching?   Is there a statust hat could be emitted that means "disabled"?